engine/job.go

```diff
@@ -45,7 +39,7 @@ type Job struct {
 
 // Run executes the job and blocks until the job completes.
 // If the job fails it returns an error
-func (job *Job) Run() error {
+func (job *Job) Run() (err error) {
```

Comment:
@LK4D4  named return is just to handle the eventual returned error in the defer func
https://github.com/docker/docker/pull/11805/files#diff-9166292813e13eaf0c52178aff024b6eR66